src/containers/SubjectPage/components/SubTopic.jsx

```diff
@@ -49,12 +49,19 @@ const SubTopic = ({
   }
 
   const topic = data.topic;
-  const subTopics = topic.subtopics.map(item => ({
-    id: item.id,
-    label: item.name,
-    selected: item.id === subSubTopicId,
-    url: toTopic(subjectId, filterIds, topicId, topic.id, item.id),
-  }));
+  const topicpath = topic.path
+    .split('/')
+    .slice(2)
+    .map(id => `urn:${id}`);
+  const subTopics = topic.subtopics.map(item => {
+    const path = [...topicpath, item.id];
+    return {
+      id: item.id,
+      label: item.name,
+      selected: item.id === subSubTopicId,
+      url: toTopic(subjectId, filterIds, ...path),
+    };
+  });
```

Comment:
Dette fikser problemet som er spesifisert [her](https://trello.com/c/HJafG3vv/666-underemne-tar-plassen-til-hovedemne-dersom-jeg-klikker-to-ganger-på-underemnet-under-underemnet-igjen). I gammel versjon av koden, for subsubtopic sendtes både topicId og topic.id som er det samme og dermed kom denne dobbelt. Effekten av dette kan sjekkes på [denne sida](https://staging.ndla.no/nn/subjects/subject:41/topic:1:1:186646/topic:1:1:186195/?filters=urn:filter:b5156afd-e11c-4426-a3ba-a1e25c03f489) der urlen til underemnet "Undervisningsfilmer..." viser dobbel topic.